shub/copy_eggs.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+import os
+from six.moves.urllib.parse import urljoin
+import click
+import requests
+
+from shub.config import get_target
+from shub.fetch_eggs import fetch_eggs
+from shub.utils import decompress_egg_files, _deploy_dependency_egg
+
+SHORT_HELP = "Sync eggs from one project with other project"
+
+HELP = SHORT_HELP + """
+
+Fetch all eggs from one project and upload them to other project. This allows
+you to easily clone requirements from old project if you set up some new
+project in dash.
+"""
+
+
+@click.command(help=HELP, short_help=SHORT_HELP)
+@click.argument("target", required=False, default='default')
+@click.argument("new_project", required=True)
```

Comment:
I think putting a required argument after a non-required argument doesn't work as we want. It'll interpret `shub copy-eggs 123` as `target=123` and `new_project=not defined`.